packages/block-library/src/navigation/style.scss

```diff
@@ -312,6 +312,11 @@ button.wp-block-navigation-item__content {
 	cursor: pointer;
 }
 
+// Flip submenu arrow icon when expanded.
+.wp-block-navigation-submenu__toggle[aria-expanded="true"] {
+	transform: scaleY(-1);
```

Comment:
@shrivastavanolo I'm curious, is there any reason you are using `scaleY` instead of ´rotate´? 

I agree that this is a better experience and that we should include this in the plugin but I'd love to know why you went with `scaleY` :)

Reply:
Thanks so much :) Approved from my end!